docs/src/user-guide/core-clp-s.md

```
@@ -125,3 +155,5 @@ compressed data:**
   the same file.
 * In addition, there are a few limitations, related to querying arrays, described in the search
   syntax [reference](reference-json-search-syntax).
+
+[1]: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-query-string-auth.html
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)